jsonschema-validation.xml

```diff
@@ -410,13 +410,13 @@
                         then this keyword has no effect.
                     </t>
                     <t>
-                        An instance array is valid against "maxContains" in two ways, depending on
-                        the form of the annotation result of an adjacent
+                        An instance array or object is valid against "maxContains" in two ways,
+                        depending on the form of the annotation result of an adjacent
                         <xref target="json-schema">"contains"</xref> keyword. The first way is if
                         the annotation result is an array and the length of that array is less than
                         or equal to the "maxContains" value. The second way is if the annotation
-                        result is a boolean "true" and the instance array length is less than or
-                        equal to the "maxContains" value.
+                        result is a boolean "true" and the instance length is less than or equal to
```

Comment:
Can "length" be applied to objects?  Maybe we keep "instance array length" and add "or property count"?

Reply:
I think "length" makes sense. It may be an object, but in this context, it's being treated like a map, which does have a concept of length. But I you think it's questionable, others will be confused, so we should be more explicit.

Reply:
"number of properties" is explicit and doesn't depend on a language/framework's particular data model.